src/getAllSoldHomeZillowURLs.py

```diff
@@ -50,7 +50,7 @@ def getAllSoldHomeZillowURLs(NorthBoundary, SouthBoundary, EastBoundary, WestBou
                                        WestBoundary)
 
             # Using the ZillowURL, its HTML is requested and parsed using getHTML function
-            ZillowHTML = getHTML(ZillowURL)
+            ZillowHTML = getHTML(ZillowURL, RequestHeaders)
```

Comment:
wow. did you really forget to do this in last pull request? sloppy codemanship there @AmritPatel